From: Filippo Giunchedi <fgiunchedi@wikimedia.org>
Date: Thu, 16 Oct 2014 11:44:09 +0100
Subject: numa.py: use /usr/bin/env

---
 src/collectors/numa/numa.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/collectors/numa/numa.py b/src/collectors/numa/numa.py
index 59c5860..3bd4e7a 100644
--- a/src/collectors/numa/numa.py
+++ b/src/collectors/numa/numa.py
@@ -1,4 +1,4 @@
-#!/usr/local/rnt/bin/python
+#!/usr/bin/env python
 # coding=utf-8
 """
 This class collects data on NUMA utilization
