From: Filippo Giunchedi <fgiunchedi@wikimedia.org>
Date: Tue, 2 Dec 2014 18:12:45 +0000
Subject: remove distro detection from setup.py

Change-Id: I7d34d744a14c064523ad2db58001702d057799a8
---
 setup.py | 17 -----------------
 1 file changed, 17 deletions(-)

diff --git a/setup.py b/setup.py
index 2c7c681..ddea2ea 100755
--- a/setup.py
+++ b/setup.py
@@ -61,23 +61,6 @@ else:
         data_files.append(('/etc/diamond/handlers',
                            glob('conf/handlers/*')))
 
-        if distro == 'Ubuntu':
-            data_files.append(('/etc/init',
-                               ['debian/upstart/diamond.conf']))
-        if distro in ['centos', 'redhat', 'debian', 'fedora']:
-            data_files.append(('/etc/init.d',
-                               ['bin/init.d/diamond']))
-            data_files.append(('/var/log/diamond',
-                               ['.keep']))
-            if distro_major_version >= '7' and not distro == 'debian':
-                data_files.append(('/usr/lib/systemd/system',
-                                   ['rpm/systemd/diamond.service']))
-            elif distro_major_version >= '6' and not distro == 'debian':
-                data_files.append(('/etc/init',
-                                   ['rpm/upstart/diamond.conf']))
-
-    # Support packages being called differently on different distros
-
     # Are we in a virtenv?
     if running_under_virtualenv():
         install_requires = ['ConfigObj', 'psutil', ]
